src/ir/wrap.jl

```diff
@@ -38,7 +38,11 @@ function IRCode(ir::IR)
         x = get(defs, br.args[1], br.args[1]) |> unvars
         push!(stmts, ReturnNode(x))
       elseif br == unreachable
-        push!(stmts, ReturnNode())
+        @static if VERSION >= v"1.11.0-DEV.655"
```

Comment:
```suggestion
        @static if VERSION >= v"1.10"
```
I tagged https://github.com/JuliaLang/julia/pull/51715 for backporting, so once 1.10 is officially released it should ship this fix as well

Reply:
Perfect!